### PR TITLE
Add Indonesian Translation for Cloud Controller Manager Concepts page

### PR DESCRIPTION
**Pull request by phanama**
This PR translates Cloud Controller Manager Concepts docs to Bahasa Indonesia for #15341 
Requesting reviews @girikuncoro @irvifa

### CODE REVIEW

File: content/id/docs/concepts/architecture/cloud-controller.md [hunk 1]
```diff
@@ -0,0 +1,238 @@
...
+* List
+* Get
+* Watch
+* Patch
+* Update
+
+### Akses Lainnya
+
+Implementasi dari inti CCM membutuhkan akses untuk membuat Event, dan untuk memastikan operasi yang aman, dibutuhkan akses untuk membuat ServicAccount.
+The implementation of the core of CCM requires access to create events, and to ensure secure operation, it requires access to create ServiceAccounts.
```
Comment (girikuncoro) [inline content/id/docs/concepts/architecture/cloud-controller.md]: forgot to throw english sentence
```suggestion

```

**Comment by girikuncoro** [comment]
@phanama beautiful translation! there are some minor typos, but overall great job! can merge after the typos have been addressed

/lgtm

File: content/id/docs/concepts/architecture/cloud-controller.md [hunk 2]
```diff
@@ -0,0 +1,238 @@
...
+* Route Controller
+* Service Controller
+
+{{< note >}}
+Volume Controller secara sengaja tidak dipilih sebagai bagian dari CCM. Hal ini adalah karena kerumitan untuk melakukannya, dan mempertimbangkan usaha-usaha yang sedang berlangsung untuk memisahkan logika volume yang spesifik vendor dari KCM, sehingga diputuskan bahwa Volume Contoller tidak akan dipisahkan dari KCM ke CCM.
+{{< /note >}}
+
+Rencana awal untuk mendukung volume menggunakan CCM adalah dengan menggunakan Flexvolume untuk mendukung penambahan volume secara _pluggable_. Namun, ada sebuah usaha lain yang diberi nama Container Storage Interface (CSI) yang sedang berlangsung untuk menggantikan Flexvolume.
+
+Mempertimbangkan dinamika tersebut, kami memutuskan untuk mengambil tidakan sementara hingga CSI siap digunakan.
```
[the remark this answers is not in the document]
Comment (phanama) [reply content/id/docs/concepts/architecture/cloud-controller.md]: Resolved in https://github.com/kubernetes/website/pull/15335/commits/6bc040f39df16dd106905e21ec9f048d262bd41c

File: content/id/docs/concepts/architecture/cloud-controller.md [hunk 3]
```diff
@@ -0,0 +1,238 @@
...
+
+* List
+* Get
+* Watch
+* Patch
+* Update
+
+### Akses Lainnya
+
+Implementasi dari inti CCM membutuhkan akses untuk membuat Event, dan untuk memastikan operasi yang aman, dibutuhkan akses untuk membuat ServicAccount.
```
[the remark this answers is not in the document]
Comment (phanama) [reply content/id/docs/concepts/architecture/cloud-controller.md]: Resolved in https://github.com/kubernetes/website/pull/15335/commits/6bc040f39df16dd106905e21ec9f048d262bd41c

File: content/id/docs/concepts/architecture/cloud-controller.md (hunk 1 above)
Comment (phanama) [reply content/id/docs/concepts/architecture/cloud-controller.md]: Resolved in https://github.com/kubernetes/website/pull/15335/commits/6bc040f39df16dd106905e21ec9f048d262bd41c

**Comment by girikuncoro** [comment]
Thanks for your contribution!
/lgtm
/approve

**Pull request merged**